pkg/utils/utils.go

```diff
@@ -19,6 +19,45 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+//ProvisionState is the way a brick or all bricks are provisioned
```

Comment:
Utils package is not a good place for this

Why not volume package ?

Reply:
ProvisionState is also used in brickinfo, that is why I decided to define in utils

Reply:
In that case, there can be two separate types - one in brick package and one in volume package.